collectors/bzimport/collectors.py

```diff
@@ -242,57 +242,80 @@ def get_product_with_components(self, name):
         return products[0] if products else None
 
 
+# TODO tzinfo should be set implicitly but my current
+# attempts do not work - leaving explicit for now
+TIMEZONE = timezone.get_current_timezone()
+
+
 class FlawCollector(Collector, BugzillaQuerier, JiraQuerier):
     """Bugzilla flaw collector"""
 
     # date before the first flaw was created
-    BEGINNING = timezone.datetime(2000, 1, 1, tzinfo=timezone.get_current_timezone())
+    BEGINNING = timezone.datetime(2000, 1, 1, tzinfo=TIMEZONE)
```

Comment:
Bugzilla ignores timezone at this time anyway. However, the documented format is "%Y-%m-%dT%H:%M:%SZ", so I think if you want to be 100% compliant with BZ API documentation, you should probably convert datetime to string before sending it to BZ library.

Reply:
I think you actually do not want any other timezone than UTC.

Reply:
The timezone stuff is not much to be review here :smile: nothing functionally changes in it, I was just moving furniture around. The timezones are more for the Python reasons like not compare timezone-aware and timezone-naive timestamps. I am not sure about the format here, I thought the magic is done by the **bugzilla**. Anyway again nothing which would be changed in this PR.